charts/shoot-core/charts/coredns/values.yaml

```diff
@@ -7,11 +7,7 @@ service:
   domain: cluster.local in-addr.arpa ip6.arpa
   port: 53
   targetPort: 8053
-  annotations:
```

Comment:
You need to add an `{{if}}{{end}}` block around line 7 in the `coredns-service.yaml` file, otherwise this error occurs:

```
time="2018-10-05T08:00:00+02:00" level=error msg="render error in \"shoot-core/charts/coredns/templates/coredns-service.yaml\": template: shoot-core/charts/coredns/templates/coredns-service.yaml:7:33: executing \"shoot-core/charts/coredns/templates/coredns-service.yaml\" at <toYaml>: wrong number of args for toYaml: want 1 got 0"
```